library/Network/CloudSeeder/Main.hs

```diff
@@ -114,11 +124,16 @@ cli (DeployStack nameToDeploy) config = do
 
   templateBody <- readFile $ nameToDeploy <> ".yaml"
 
+  let decodeOrFailure = decodeEither (encodeUtf8 templateBody) :: Either String Value
```

Comment:
Is this `Either String Value` type annotation necessary? Is it because the aeson lenses are generic, so we have to pick `Value` instead of some stringy type?